index.js

```diff
@@ -0,0 +1,44 @@
+
+
+// First exercise --------------------
+const countMovesUp = inputValue => inputValue.match(/\(/g).length
+
+const countMovesDown = inputValue => inputValue.match(/\)/g).length
+
+const santasFloor = inputValue => countMovesUp(inputValue) - countMovesDown(inputValue)
+
+
+// Second exercise --------------------
+const santasFirstPositionAtSubsolo = ( position, santasFlorPosition) => {
+  let i = position;
+
+  for ( i; i < inputValue.length; i++ ) {
+    if ( inputValue[i] == '(' ) {
+      santasFlorPosition += 1
+    } else {
+      santasFlorPosition -= 1
+      if ( santasFlorPosition == -1 ) {
```

Comment:
I think you could remove this `if` from inside the `else`. It will have the same result and will be more readable. And maybe comparison `santasFlorPosition < 0` is safetier than `santasFlorPosition == -1`